### DISCUSSION TITLE
Projeto inicial

### DISCUSSION THREAD

**Discussion by williamWHC** [williamWHC/Rocketseat#3] [Q&A] [answered]
A estrutura de pastas está de acordo com os padrões ?

**Comment by williamWHC** [comment]
Como está arquivo index.html apresentado ?

**Reply by williamdoxv** [reply]
Está de acardo podemos fechar assim.

**Comment by williamdoxv** [comment, marked as answer]
Está de acrodo com os padrões, pode seguir assim.
